csharp/ql/test/library-tests/dataflow/typeflow-dispatch/TypeFlowDispatch.cs

```diff
@@ -0,0 +1,104 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public class A
+{
+    static T Source<T>(T source) => throw null;
+
+    public static void Sink<T>(T t) { }
+
+    interface MyConsumer
```

Comment:
This looks unused, right? Looks like a leftover from the conversion from Java. Otherwise LGTM now.

Reply:
Yes, that was not meant to be included. Removed now.